velox/substrait/TypeUtils.h

```diff
@@ -14,11 +14,15 @@
  * limitations under the License.
  */
 
+#include "velox/substrait/SubstraitParser.h"
 #include "velox/type/Type.h"
 
 namespace facebook::velox::substrait {
 
 /// Return the Velox type according to the typename.
 TypePtr toVeloxType(const std::string& typeName);
 
+// Get Expression from FunctionArgument
+const ::substrait::Expression& getExprFromFunctionArgument(
```

Comment:
This function doesn't have anything to do with types, hence, it doesn't belong to a file named TypeUtils.h.

I wonder if we can just remove this function and change the callers to use arg.value() directly.